web-ng/src/app/app.module.ts

```diff
@@ -72,6 +77,12 @@ const firebaseUiAuthConfig: firebaseui.auth.Config = {
     UserProfilePopupModule,
     LayerDialogModule,
     ShareDialogModule,
+    MatInputModule,
+    LayerDialogModule,
+    MatCardModule,
+    CardViewProjectModule,
+    MatGridListModule,
+    ShareDialogModule,
```

Comment:
Can these modules be included in the module where they're used rather than in the global app module?

Reply:
Gentle ping.

Reply:
@HemanParbhakar ?

Reply:
will remove the ones.